hw/mcu/nordic_sdk/pkg.yml

```diff
@@ -51,9 +51,9 @@ pkg.src_dirs:
     - "src/ext/nRF5_SDK_11.0.0_89a8197/components/drivers_nrf/"
     - "src/ext/nRF5_SDK_11.0.0_89a8197/components/libraries/fifo/"
     - "src/ext/nRF5_SDK_11.0.0_89a8197/components/libraries/util/"
-
+    #- "src/ext/nRF5_SDK_11.0.0_89a8197/components/libraries/pwm/"
```

Comment:
Does this need to be conditionally included in src_dirs?
I assume conditional include like
`pkg.src_dirs.PWM:`
`   - "src/ext/nRF5_SDK_11.0.0_89a8197/components/libraries/pwm/"`
should work. If not, we should add that to newt tool.